Disaster Recovery/SafeHaven 4/Deploy-CMS-and-SRN-in-CenturyLink-Cloud.md

```diff
@@ -0,0 +1,98 @@
+{{{
+  "title": "SafeHaven-4-Deploy CMS and SRN in CenturyLink Cloud",
+  "date": "05-04-2016",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+## Article Overview
+This article explains how to deploy the Central Management Server(CMS) and SafeHaven Replication Node(SRN) in CenturyLink Cloud.
+
+A single CMS server is required for a single SafeHaven Cluster. Typically CMS resides in the DR datacenter so that users can access it and manage their SafeHaven environment, even if the production datacenter is unavailable. In this scenario our DR site is CA2-Canada(Toronto) in CenturyLink Cloud.
+
+SRN resides in both production and DR datacenters and work in pairs. A single SafeHaven Cluster can have upto 64 SRN's registered. Total number of SRN's depend on the size and type of environment that needs to be protected. 
+
+Please refer to the **Support Matrix** section in the [SafeHaven 4.0.1 Release Notes](safehaven-4.0.1-release.md) for more information.
+
+### Requirements
+
+Login access to the CenturyLink Cloud Portal at https://control.ctl.io.
+
+### Assumptions
+
+This article assumes that the user has login access to the CenturyLink Cloud Portal. Login to the **CenturyLink Control Portal**  https://control.ctl.io with your credentials.
+
+![Upgrade](../../images/SH4.0/Create%20Nodes/Nodes1.png)

```

Comment:
Change link to `../../images/SH4.0/Create Nodes/Nodes1.png`
Likewise change the links below that contain `Create%20Nodes`
These are expect to be filenames, not URLs.